tools/dep_updaters/update-libuv.sh

```diff
@@ -2,16 +2,29 @@
 set -e
 # Shell script to update libuv in the source tree to a specific version
 
-BASE_DIR=$(cd "$(dirname "$0")/../.." && pwd)
-DEPS_DIR="$BASE_DIR/deps"
-LIBUV_VERSION=$1
-
 if [ "$#" -le 0 ]; then
-  echo "Error: please provide an libuv version to update to"
+  echo "Error: please provide an libuv version to update"
```

Comment:
Was this edit done by mistake? The original sentence makes more sense to me.
```suggestion
  echo "Error: please provide an libuv version to update to"
```

Reply:
Yeah, I initially modified the sentence and then reverted it manually without taking care of the diff :/